app/controllers/api/v1/items/find_items_controller.rb

```diff
@@ -14,4 +15,10 @@ def item_params
     params.permit(:id, :description, :name, :unit_price, :merchant_id, :created_at, :updated_at)
   end
 
+  def format_unit_price
```

Comment:
I like it here. I would consider `integer` equally 'correct' in our database (and maybe even more so that we don't have to deal with any rounding surprises in our stored data). I think it makes sense to have a method like this primarily concerned with how we're presenting our data.